src/App.css

```diff
@@ -62,7 +72,96 @@ body {
 
 #root {
   display: flex;
+  flex-direction: column;
   align-content: center;
   justify-content: center;
   align-items: center;
 }
+
+nav > ul {
+  display: flex;
+  flex-direction: row;
+  gap: 10px;
+  list-style: none;
+  padding: 0;
+}
+
+/* ul > li {
+  border-right: 2px solid black;
+  border-left: 2px solid black;
+  padding-left: 10px;
+  padding-right: 10px; */
+
+/* } */
+
```

Comment:
- [ ] Please make sure you remove commented code from your project to make your code clean. :pray: You should not push commented code to your repository. You can read more about it [here](https://www.freecodecamp.org/news/code-comments-the-good-the-bad-and-the-ugly-be9cc65fbf83/)